problemsets-mat223/modules/module4-exercises.tex

```diff
@@ -1,17 +1,124 @@
+		\end{enumerate}
+		\prob For each vector, find two \emph{unit} vectors orthogonal to it
+		\begin{enumerate}
+			\item $\mat{0\\1}$
+			\item $\mat{1\\2}$
+			\item $\mat{1\\3\\5}$
+			\item $\mat{-13\\-4\\5}$
+			\item $\mat{0\\1\\1\\\frac{1}{2}}$
+		\end{enumerate}
+
+		\prob Compute the distance between the following pairs of vectors.
+		\begin{enumerate}
+			\item $\mat{-1\\1}$ and $\mat{-1\\-4}$
+			\item $\mat{2\\-6\\5}$ and $\mat{-4\\7\\-3}$
+			\item $\mat{1\\1\\1}$ and $\mat{-1\\-1\\-1}$
+			\item $\mat{6\\0\\2\\1}$ and $\mat{0\\1\\3\\-1}$
+			\item $\mat{0\\0\\0\\0\\0}$ and $\mat{1\\1\\1\\1\\1}$
+		\end{enumerate}
+
+		\prob
+		\begin{enumerate}
+			\item Which vector out of $\mat{1\\0}$, $\mat{0\\1}$, and $\mat{4\\1}$
+			has a direction closest to $\mat{3\\5}$?
+			\item Which vector out of $\mat{2\\3\\4}$, $\mat{1\\-1\\-1}$, and $\mat{-3\\0\\1}$
+			has a direction closest to $\mat{1\\0\\1}$?
+		\end{enumerate}
+
+		\prob For each plane specified, express the plane in both vector form and normal form.
+		\begin{enumerate}
+			\item The plane $\mathcal P$ passing through the points
+			$A=(2,0,0)$, $B=(0,3,0)$ and $C=(0,0,-1)$.
+			\item The plane $\mathcal Q$ passing through the points
+			$D=(1,1,1)$, $E=(1,-2,1)$ and $F=(0,12,0)$.
+		\end{enumerate}
+
+		\prob
+		\begin{enumerate}
+			\item Let $\mathcal A\subseteq \R^3$ be the plane passing through $\mat{0\\1\\1}$
+			and with normal vector $\mat{-1\\-1\\-1}$. Write $\mathcal A$ in vector form.
+			\item Let $\mathcal B\subseteq \R^3$ be the plane passing through $\mat{1\\2\\3}$
+			and with normal vector $\mat{1\\-1\\0}$. Write $\mathcal B$ in vector form.
+		\end{enumerate}
+
+		% Removed Question
+
+		% \prob Find unit vectors in the same direction as the following vectors.
+		% \begin{enumerate}
+		% 	\item $\mat{90\\0}$
+		% 	\item $\mat{16\\4}$
+		% 	\item $\mat{11\\0\\13}$
+		% 	\item $\mat{3\\5\\9}$
+		% 	\item $\mat{0\\1\\0\\-1}$
+		% 	\item $\mat{434\\1337\\1999\\4568}$
+		% \end{enumerate}
+
+		\prob
+		\begin{enumerate}
+
+			\item Show by direct computation
+
+			$\left(\mat{1\\2} + \mat{3\\4}\right) \cdot \mat{5\\6} =
+			\mat{1\\2} \cdot \mat{5\\6} + \mat{3\\4} \cdot \mat{5\\6}$
+
+			\item
+			For $x,y,z\in R^2$, is it always true that
+
+			$(\vec x + \vec y) \cdot \vec z = \vec x \cdot \vec z + \vec y \cdot \vec z$?
 
+			Justify your answer.
+
+			\item
+			For $x,y,z\in R^n$, is it always true that
+
+			$(\vec x + \vec y) \cdot \vec z = \vec x \cdot \vec z + \vec y \cdot \vec z$?
+
+			Justify your answer.
+
+			\item Show by direct computation
+
+			$\left(6\mat{2\\3}\right) \cdot \mat{4\\5} =
+			6\left(\mat{2\\3} \cdot \mat{4\\5}\right)$
+
+			\item
+			For $x,y\in R^2$ and $k \in \R$ is it always true that
+			$(k\vec x) \cdot \vec y = k(\vec x \cdot \vec y)$?
+			Justify your answer.
+
+			\item
+			For $x,y\in R^n$ and $k \in \R$ is it always true that
+			$(k\vec x) \cdot \vec y = k(\vec x \cdot \vec y)$?
+			Justify your answer.
```

Comment:
+ `\item The dot product is called \emph{distributive}. Is this a good word to describe the dot product? Why?`